framework/src/media/utils/MediaUtils.cpp

```diff
@@ -307,24 +307,35 @@ bool header_parsing(FILE *fp, audio_type_t audioType, unsigned int *channel, uns
 	unsigned char *header;
 	unsigned char tag[2];
 	bool isHeader;
+	int ret;
 
 	switch (audioType) {
 	case AUDIO_TYPE_MP3:
 		isHeader = false;
-		while (fread(tag, sizeof(unsigned char), 1, fp) > 0) {
-			if (tag[0] == 0xFF) {
+		while (fread(tag, sizeof(unsigned char), 2, fp) == 2) {
```

Comment:
If fread failed because of internal problem, we should stop below all
```
ssize_t ret;
while(1) {
    ret = fread();
    if (ret != 2) {
        error_log(errno());
        return ERROR;
    }
    if (check_sync_word) {
         is_header_true;
    } else {
         seek();
   }
}
```

Reply:
you right.
but, many places use fseek and fread. so, i will cleanup later.

Reply:
got it. please do not forget it.

Reply:
yes